app/src/androidTest/kotlin/cz/covid19cz/erouska/screenObject/FinishActivation.kt

```diff
@@ -0,0 +1,25 @@
+package cz.covid19cz.erouska.screenObject
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import cz.covid19cz.erouska.R
+import org.awaitility.Awaitility.await
+import java.util.concurrent.TimeUnit
+
+class FinishActivation {
+    companion object {
+        const val TITLE = "eRouška potřebuje běžet i když s ní právě nepracujete"
+    }
+
+    fun finish() {
+
+        await().ignoreExceptions().atMost(5, TimeUnit.SECONDS).untilAsserted {
```

Comment:
Tyhle waity by to chtělo nahradit Idling Resources viz https://developer.android.com/training/testing/espresso/idling-resource

Testy jsou pak rychlejší a stabilnější. Ale vyžaduje to zásah i do kódu aplikace, otázka jestli se vám to chce dělat. Ale v profi testech by waity neměly být.